### ISSUE TITLE
Help for making it work

### ISSUE THREAD

**Issue by gbip** [pegasos1/cargo-profiler#34]
Hello,
I can't figure out how to profile my binary.
I usually start it like that ```target/debug/mybin -r my/path -w something.png```.

I tried call cargo-profiler like this : ```cargo profiler callgrind -n 15 --release -- -r scenes/suzanne_low.json -- -w test.png```

I also tried to specify the binary by hand : ``` cargo profiler callgrind -n 10 --bin target/release/render_engine -- -r scenes/suzanne_low.json -- -w prof.png ```

The problem is that in both cases, the program doesn't parse any arguments at all : it silently stop at roughly 600k instructions.
Can you explain to me how should I pass arguments to the binary ?
Thanks a lot !

**Comment by beamspease** [comment]
I too seem to be having this problem

**Comment by gbip** [comment]
Yep but it seems that this tool is not actively develloped.

**Comment by dylanmckay** [comment]
Also having this problem

**Comment by peterbraden** [comment]
It appears that argument parsing isn't working at all.

In the readme argument parsing is v0.1.6 whereas the Cargo.toml refers to 0.1.5 - perhaps the latest version isn't published?

It appears installing from github works:
`cargo install cargo-profiler --git http://git@github.com/kernelmachine/cargo-profiler.git`

**Comment by osa1** [comment]
Can confirm that installing from github works.

Too bad this crate is not maintained -- it's a really useful one.

**Comment by sourcefrog** [comment]
Yes, @peterbraden 's suggestion seems to work for me. It'd be great if there was a new release to Crates.io with the fix...
